### ISSUE TITLE
Players can Rename Tripwire Hooks as Crate Keys and Use Them

### ISSUE THREAD

**Issue by Zenoxez** [ConnorLinfoot/CratesPlus#304]
**Please use the issue template below. Issues not using the templates will likely be closed without warning! If you are suggesting a feature you do not need to follow the template.**

If you're having config issues (such as it resetting) first make sure it's a valid YAML file using a [parser](http://yaml-online-parser.appspot.com).

From version 4.2 there is a new '/crates debug' command. Running this will give you a single link with all the data I need to help you.

### Link/paste of stack trace/error (if any)

    You can also simply just use code formatting in markdown!
___We need all of the stack trace! Do not cut off parts of it.___

### Debug Link
___Enter your debug link from /crates debug___

### Actions to reproduce (if known):
___This may include a build schematic, a video, or detailed instructions to help reconstruct the issue___

Basically, players can make a Tripwire hook, rename it to the name of a Crate Key in an anvil, and then use it on crates... This is very obnoxious.

**Comment by Zenoxez** [comment]
Bump - This is a major issue

**Comment by ConnorLinfoot** [comment]
I've been unable to replicate this, as long as you don't have plugins modifying how anvils work or modified the key name to something without the crate name this shouldn't be possible.

**Comment by Zenoxez** [comment]
What do you mean exactly by "modified the key name to something without the crate name?"
